libgloss/arc/stub.c

```diff
@@ -33,3 +33,16 @@ _close (int fd)
 {
     return 0;
 }
+
```

Comment:
Commit against ARCv1.v2 backend. Please remove it.

Reply:
Dropped everything related to arcV2. Thanks!